src/filesystem/impls/filer/FilerFileSystem.js

```diff
@@ -158,7 +159,26 @@ define(function (require, exports, module) {
     }
 
     function rename(oldPath, newPath, callback) {
-        fs.rename(oldPath, newPath, _wrap(callback));
+        function updateBlobURL(err) {
+            if(err) {
+                return callback(err);
```

Comment:
I think all your callbacks where you return an error, you have to called `_mapError(err)` like you do below in line 238